2022-06-22 07:58:08,129 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2022-06-22 07:58:08,133 INFO supervisord started with pid 8
2022-06-22 07:58:09,136 INFO spawned: 'nginx' with pid 11
2022-06-22 07:58:09,139 INFO spawned: 'app' with pid 12
2022-06-22 07:58:10,157 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-06-22 07:58:10,157 INFO success: app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
Bottle v0.12.21 server starting up (using WSGIRefServer())...
Listening on http://localhost:5000/
Hit Ctrl-C to quit.

[CTRL] POST /setup
[Time: 22.06.22 07:58:23] [Level: info] id: 03767e1965785540
[Time: 22.06.22 07:58:23] [Level: info] coordinator: False
[Time: 22.06.22 07:58:23] [Level: info] clients: ['e312397f3387a197', '71703185f2e02ae5', '01148f16bc35f1a8', 'c3692abf23407a71', '03767e1965785540', 'f927577bf47184e9', 'e152f72648bd2a5c', '46846284a3ca5830']
[Time: 22.06.22 07:58:23] [Level: info] state: initial
[Time: 22.06.22 07:58:23] [Level: info] transition: read
127.0.0.1 - - [22/Jun/2022 07:58:23] "POST /api/setup HTTP/1.0" 200 0
[CTRL] GET /status
127.0.0.1 - - [22/Jun/2022 07:58:23] "GET /api/status HTTP/1.0" 200 126
[Time: 22.06.22 07:58:24] [Level: info] state: read
[Time: 22.06.22 07:58:24] [Level: info] transition: await
[Time: 22.06.22 07:58:25] [Level: info] state: await
[CTRL] GET /status
127.0.0.1 - - [22/Jun/2022 07:58:26] "GET /api/status HTTP/1.0" 200 126
[CTRL] GET /data
127.0.0.1 - - [22/Jun/2022 07:58:26] "GET /api/data HTTP/1.0" 200 15297
[CTRL] POST /data
[CTRL] GET /status
127.0.0.1 - - [22/Jun/2022 07:58:29] "POST /api/data?client=e312397f3387a197 HTTP/1.0" 200 0
127.0.0.1 - - [22/Jun/2022 07:58:29] "GET /api/status HTTP/1.0" 200 127
[Time: 22.06.22 07:58:29] [Level: info] Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/FeatureCloud/app/engine/app.py", line 169, in guarded_run
    self.run()
  File "/usr/local/lib/python3.8/site-packages/FeatureCloud/app/engine/app.py", line 184, in run
    transition = self.current_state.run()
  File "/app/states.py", line 86, in run
    c.delete_columns(to_drop=to_drop)
  File "/app/states.py", line 23, in delete_columns
    self.df = self.df.drop(columns=to_drop)
  File "/usr/local/lib/python3.8/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/pandas/core/frame.py", line 4954, in drop
    return super().drop(
  File "/usr/local/lib/python3.8/site-packages/pandas/core/generic.py", line 4267, in drop
    obj = obj._drop_axis(labels, axis, level=level, errors=errors)
  File "/usr/local/lib/python3.8/site-packages/pandas/core/generic.py", line 4311, in _drop_axis
    new_axis = axis.drop(labels, errors=errors)
  File "/usr/local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 6640, in drop
    indexer = self.get_indexer_for(labels)
  File "/usr/local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 5764, in get_indexer_for
    return self.get_indexer(target)
  File "/usr/local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 3784, in get_indexer
    return self._get_indexer(target, method, limit, tolerance)
  File "/usr/local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 3809, in _get_indexer
    indexer = self._engine.get_indexer(tgt_values)
  File "pandas/_libs/index.pyx", line 305, in pandas._libs.index.IndexEngine.get_indexer
  File "pandas/_libs/hashtable_class_helper.pxi", line 5247, in pandas._libs.hashtable.PyObjectHashTable.lookup
TypeError: unhashable type: 'Index'

